### PR TITLE
connect firebase to the project + added functioanilty to write data in the data base

### LINKED ISSUE #32: connect firebase to the project

### PR DESCRIPTION
**Pull request by Alez05**
Fixes #32

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+#compiled output
+.env
+
```
Comment (marianzburlea) [inline .gitignore:5]: if you also use `.env.local` you want to add it too
otherwise ignore

File: src/app/server/firebase.server.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      clientEmail: process.env.FIREBASE_CLIENT_EMAIL,
+      privateKey: process.env.FIREBASE_PRIVATE_KEY?.replace(/\\n/g, '\n'),
+    } as admin.ServiceAccount),
+  });
+}
+
+
+// write in database - this function will write data in database
+
+export const writeData = async (table = '', date={}, id ='' ) => {
```
Comment (marianzburlea) [inline src/app/server/firebase.server.ts]: date -> `data`
or `info`

**Review by marianzburlea — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (Alez05) [reply .gitignore:5]: I only made use of .env so far

File: src/app/server/firebase.server.ts (hunk 2 above)
Comment (Alez05) [reply src/app/server/firebase.server.ts]: fixed;

**Review by florin-bizgan — APPROVED** [review]
Looks good

**Pull request merged**